.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+(please ensure you are running the latest version of Cloud Tools for IntelliJ with _Help > Check for Updates_.)
```

Comment:
Maybe add a note that issues such as feature requests etc don't strictly have to follow this template.